src/instrumentsscene/view/layoutpaneltreemodel.cpp

```diff
@@ -828,6 +862,13 @@ void LayoutPanelTreeModel::updateMovingDownAvailability(bool isSelectionMovable,
     bool hasControlItem = parentItem->type() != LayoutPanelItemType::ROOT;
     const AbstractLayoutPanelTreeItem* curItem = modelIndexToItem(lastSelectedRowIndex);
     bool lastSelectedIsSystemObjectLayer = curItem && curItem->type() == LayoutPanelItemType::ItemType::SYSTEM_OBJECTS_LAYER;
+
+    IF_ASSERT_FAILED(!(lastSelectedIsSystemObjectLayer && lastSelectedRowIndex.row() == 0)) {
```

Comment:
This is a bit hard to read: you have to do a few logical inversions in your head